src/containers/auth/register/index.js

```diff
@@ -21,6 +22,19 @@ class Register extends Component {
     errors: {},
   };
 
+  componentDidMount() {
```

Comment:
if you destructed location and history, why did you not destruct googleLoginRequest and linkedInLoginRequest from props?